src/languages/en-US.ts

```diff
@@ -1381,6 +1381,15 @@ export default class extends Language {
 					a temporary ban.`,
 			examples: ['@Pete', '@Pete Spamming all channels.', '@Pete Spamming all channels, for 24 hours.']
 		}),
+		COMMAND_DEHOIST_DESCRIPTION: 'Shoot everyone with the dehoist-inator 3000',
+		COMMAND_DEHOIST_EXTENDED: builder.display('dehoist', {
+			extendedHelp: `The act of hoisting involves adding a character with a value of 64 and below in front of your nickname
+			in order to appear higher in the members list. This command replace's with a special character that drags them to the bottom
+			of the list. Ha, get a load of this, mr. \`! !\`
+			
+			This command requires **${PERMS.MANAGE_NICKNAMES}**, and only members with lower role hierarchy position
```

Comment:
This should go to the `reminder` option from the builder, not in `extendedHelp`.